proof/invariant-abstract/DetSchedSchedule_AI.thy

```diff
@@ -21248,9 +21287,10 @@ lemma handle_event_cur_sc_in_release_q_imp_zero_consumed:
   apply (cases e; simp)
 
   subgoal for syscall
-    by (case_tac syscall; simp
+    apply (case_tac syscall; simp
         ; cur_sc_in_release_q_imp_zero_consumed_syscall_single?
           , cur_sc_in_release_q_imp_zero_consumed_syscall_combined?) \<comment> \<open>takes 30 seconds or so\<close>
+    by fastforce+
```

Comment:
Since this has a `takes long` comment, I'd recommend to use the intial-method/final-method form of `by`, so that interactive proof progress isn't stuck at this point, i.e

```
by (case_tac ...; ..)
     fastforce+
```